oligo_designer_toolsuite/oligo_specificity_filter/_utils.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+import pandas as pd
+
+
+def _write_output(self, probes_info, gene_id, probes_wo_match):
```

Comment:
we dn't need that function if we plly the filtering directly on probes DB. But we should add an additional function in I/O to write genes without probes into a file (or genes with probes < x)